templates/documentation/permafrost.html

```diff
@@ -52,12 +52,16 @@ <h4>Geophysical Institute Permafrost Lab (GIPL) model output (point query)</h4>
   </tbody>
   <tfoot>
     <tr>
-      <td colspan="2">CSV output is also available by appending <code>?format=csv</code> to the URL.</td>
+      <td colspan="2">Min/mean/max summaries of projected years are available by appending
```

Comment:
Consider indicating how to chain these args like `/permafrost/point/gipl/63.0628/-146.1627?summary=mmm&format=csv`

Reply:
Good idea Charlie! Added with [eef8f61](https://github.com/ua-snap/data-api/pull/341/commits/eef8f613dab5e43f2bb4ff2804a4174824d6bb3f)